### PR TITLE
CPU mitigations formula

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/8914

### CODE REVIEW

File: modules/salt/pages/salt-formulas.adoc [hunk 1]
```diff
@@ -1016,3 +1018,39 @@ Then it will start the VSFTP service (``vsftpd``).
...
 
 For more information about setting up and tuning Vsftpd, see the documentation coming with the [package]``vsftpd`` package ([path]``/usr/share/doc/packages/vsftpd/`` when the package is installed).
+
+
+
+=== CPU Mitigation Formula
+
+
+CPU mitigations have been introduced to improve security on CPUs affected by vulnerabilities such as Meltdown and Spectre.
+The mitigations are available in {sles} 15 SP3 and later.
```
Comment (keichwa) [inline modules/salt/pages/salt-formulas.adoc]: I'm not sure, but 15SP3 still does not exist ;)

```suggestion
The mitigations are available in {sles} 12 SP3 and later.
```

**Comment by mseidl** [comment]
looks good out outside of my comment, if you can think of a better way to phrase it, go ahead..(wrt to my comment)

**Comment by Loquacity** [comment]
> looks good out outside of my comment, if you can think of a better way to phrase it, go ahead..(wrt to my comment)

Hi Martin, I can't see your comment?

**Comment by mseidl** [comment]
I should've made the clarification, that part wasn't part of the original text. So, it should be worded better.... and this is an option I added as a part of the formula

Maybe "Manual: If you want to set your own custom options."

**Comment by mseidl** [comment]
oh iu had to submit a review to make the comments come up. sorry i was unaware there you go...

File: modules/salt/pages/salt-formulas.adoc [hunk 2]
```diff
@@ -1016,3 +1018,39 @@ Then it will start the VSFTP service (``vsftpd``).
...
+This can be useful if you experience an L1 terminal fault side-channel problem.
+// This error will need further info. LKB 2018-08-01
+
+Off::
+All mitigations are disabled.
+This setting gives the highest performance, but compromises your security.
+Do not use this setting where untrusted code might be used.
+
+Manual::
+Which just allows the user to decide
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/salt-formulas.adoc]: Yes, that's why we're still in draft. I need a little more info about how you would go about doing this. When I had a look in the UI, I could select 'manual', but couldn't see a why to select custom settings. How would I do that?

[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/salt-formulas.adoc]: I would also like, if it's possible, a list of the various mitigations. We discuss the SMT mitigation, but nothing about any others?

[the remark this answers is not in the document]
Comment (mseidl) [reply modules/salt/pages/salt-formulas.adoc]: Manual just means there is no option there.

[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/salt-formulas.adoc]: So, in other words, manual disables all mitigations? If someone wants to enable individual ones, how would they do that?

[the remark this answers is not in the document]
Comment (mseidl) [reply modules/salt/pages/salt-formulas.adoc]: I don't know. ?  I'll see if i can find something out...

[the remark this answers is not in the document]
Comment (mseidl) [reply modules/salt/pages/salt-formulas.adoc]: There was no official suse documentation on this, so i'd leave it like that i guess?

[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/salt/pages/salt-formulas.adoc]: I found a kbase article with some help from the docs team.

**Pull request merged**